### DISCUSSION TITLE
Mission Categories Improvements

### DISCUSSION THREAD

**Discussion by Skizzles** [BG-Software-LLC/SuperiorSkyblock2#823] [Pending Feature Requests]
Couple improvements to Mission Categories:
- Ability to have a category unlock after a specific Mission is completed, or a specific category is completed.
- A placeholder we could add to a mission category that shows the requirements to unlock said category. 
- Option to allow to still view the category or to not allow opening the category until requirements are met.

This would allow you to implement quite a few different category setups like "easy, medium, hard" or "Main mission category, Side mission 1 category (Complete X mission in Main Mission to unlock), Side mission 2 category (Complete X mission in Main Mission to unlock) and have the player easily tell that a category is not available.

**Comment by OmerBenGera** [comment]
The solution for such thing as of now, is just to create missions in these categories that cannot be accessed - right?

**Reply by Skizzles** [reply]
> The solution for such thing as of now, is just to create missions in these categories that cannot be accessed - right?

Uh I believe so yes, I was more so wanting some type of way to visually see and portray unlocking a category to the player. Unsure how to accomplish that without some sort of placeholder.
